### DISCUSSION TITLE
Add Custom App/Domain to list of toggleable services

### DISCUSSION THREAD

**Discussion by jrbanach** [AdguardTeam/AdGuardHome#7296] [General]
i've been looking through the yaml but i'm not seeing it in there.  

I'd like to have a new service in the list (happykids, one of those youtube clones).  I don't want to block it outright but instead on a few devices at specific times.

is there a way we can edit that list of services on our own?

**Comment by schzhn** [comment]
Hello, @jrbanach. Currently you cannot edit the list of blocked services.
Related #1692
